cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -51,60 +51,98 @@
 
    ========================================================================== #}
 
+
+{# TODO: Programatically set the images and padding via Django #}
+{% set img=image(value.image, 'original') %}
+
+{% if value.small_image %}
+    {% set sm_img=image(value.small_image, 'original') %}
+{% else %}
+    {% set sm_img = image(value.image, 'original') %}
+{% endif %}
+
+{% set bp_sm_padding = "41.4893617%" %}
```

Comment:
Could we add a note about why these magic numbers are necessary? Does it have to do with the django note on line 55?